content/millennium/bulk-data.md

```diff
@@ -0,0 +1,91 @@
+
+In the case of Group Export, client application developers will need to work with healthcare organizations to identify the available Groups to be exported. Organizations can manage Groups within Ignite Management Tooling.
+
+_Notes_
+
+* Authentication is required, open endpoints do not support bulk export workflows.
+* Cerner supports v1.0.1 of the [FHIR Bulk Data Access (FLATFHIR) Implementation Guide](https://hl7.org/fhir/uv/bulkdata/STU1.0.1/)
+* Bulk data runs against an organization's database and utilizes the same services running other applications, so it is important to consider performance. Bulk data exports large amounts of data for large groups of patients, which takes more time to complete the larger the data set or patient population.
+
+## Client Errors
+
+The following are the possible client errors that can be received on API calls:
+
+1. Failing to send a required search parameter or sending a parameter with an invalid value will result in a `400 Bad Request` response.
+
+        HTTP/1.1 400 Bad Request
+
+2. Making a request without valid credentials will result in a `401 Unauthorized` response.
+
+        HTTP/1.1 401 Unauthorized
+
+3. Requesting data from an unknown instance or an instance where the application is not authorized will result in a `403 Forbidden` response.
+
+        HTTP/1.1 403 Forbidden
+
+4. Requesting a resource which does not exist will result in a `404 Not Found` response.
+
+        HTTP/1.1 404 Not Found
+
+5. Requesting a media type other than JSON will result in a `406 Not Acceptable` response.
+
+        HTTP/1.1 406 Not Acceptable
+
+        Content-Length: 0
+
+6. Performing an export kick-off using unsupported search parameter values
+
+        HTTP/1.1 422 Unprocessable Entity
+
+### Lenient Workflow
+
+When the optional header of `Prefer: handling=lenient` is passed to the export kick-off request, any unknown or unsupported parameters will be ignored as specified [here](http://hl7.org/fhir/uv/bulkdata/STU2/export.html#query-parameters), and the request will be processed instead of returning an error.
+
+## OperationOutcomes
+
+An [OperationOutcome](https://www.hl7.org/fhir/r4/operationoutcome.html) may be returned to provide additional context to an error.  The tables below describes common OperationOutcomes and their causes.
+
+ HTTP Status | Cause                                                                                         | Severity  | Code
+-------------|-----------------------------------------------------------------------------------------------|-----------|---------------
+ 422         | Export kick-off request contains unsupported query parameter values                           | error     | business-rule
+ 400         | Export kick-off request is missing required search parameters or has them with invalid values | error     | extension
```

Comment:
we also return operation outcome for 500 and he 429 case right?

Reply:
Updated the errors here: https://github.com/cerner/fhir.cerner.com/pull/899/commits/eff91ed66335ec7340cb5a2ec8a51663c1bd9cbf#diff-c2a4803bbbf3b17e514d41cc6eb7d578c94a283f937cce0400f5437570744d7fR114

Reply:
Made some changes.
Common OperationOutcomes: https://github.com/cerner/fhir.cerner.com/pull/899/commits/780d6472fc27b87e37ef7361457f44e8da969b69#diff-b781e82e47439c1cc68cf3a6a51ab1668b5f40d37870022bec1936c5c69d6bc1R83-R95